src/main/resources/static/mini-manage-api-stubs.yml

```diff
@@ -1296,3 +1325,23 @@ components:
       required:
         - date
         - availableBeds
+    CrnSearch:
+      type: object
+      properties:
+        crn:
+          type: string
+          example: X320741
+      required:
+        - crn
+    CrnSearchResult:
```

Comment:
This might be better expressed as a `person`. It also gives us flexibility to return more attributes later